spec/system/organization/user_views_an_organization_spec.rb

```diff
@@ -30,6 +30,16 @@
         end
       end
 
+      it "shows the right amount of articles in sidebar" do
+        expect(page).to have_content("2 posts published")
+      end
+
+      it "shows the right amount of articles when articles > 8" do
```

Comment:
There's something about the numerical values here that's making this harder to understand — we only create **7** articles, but there's somehow **9** and **8** feels a bit of a "magic number". 

Maybe it would be helpful to indicate that the value-add here is confirming that the count includes _all_ articles outside of pagination?

Reply:
Agreed. I'll move that to a place where its a lot easier to understand